docs/mast/mast.rst

```diff
@@ -313,17 +313,33 @@ Product filtering can also be applied directly to a table of products without pr
 .. code-block:: python
 
                 >>> from astroquery.mast import Observations
-                
+
                 >>> products = Observations.get_product_list('2003839997')
                 >>> print(len(products))
                 31
-                
+
                 >>> products = Observations.filter_products(data_products,
                 ...                                         productSubGroupDescription=["RAW", "UNCAL"],
                 ...                                         extension="fits")
                 >>> print(len(products))
                 4
 
+Downloading a Single File
+-------------------------
+
+You can download a single data product file using the ``Observation.download_file`` method, and passing in
+a MAST dataURL.  The default is to download the file the current working directory, which can be changed with
+the `local_path` keyword argument.
+
+.. code-block: python
+
+                >>> from astroquery.mast import Observations
+
+                >>> product = 'mast:IUE/url/pub/iue/data/lwp/13000/lwp13058.elbll.gz'
+                >>> Observations.download_file(product)
```

Comment:
This call does return a value, which is not reflected here. Suggest `result = Observations.download_file(product)`. And then adding a `print(result)`.